_episodes_rmd/05-data-structures-part2.Rmd

```diff
@@ -27,6 +34,8 @@ the data.frame - that table that we started with that information from a CSV
 gets dumped into when we load it. In this lesson, we'll learn a few more things
 about working with data.frame.
 
+##Adding columns and rows in data.frame
```

Comment:
I believe our current style guide (I *think* we have a style guide?) suggests a space after the pound signs for headers. This is how the other lessons are. Can you add a space after each pair of `##` before the heading text in the heading throughout?